components/eamxx/src/share/io/tests/io_alias.cpp

```diff
@@ -180,42 +207,192 @@ TEST_CASE("output_aliases_integration", "[.][io][alias]") {
     auto& ctrl_pl = params.sublist("output_control");
     ctrl_pl.set<std::string>("frequency_units", "nsteps");
     ctrl_pl.set<int>("frequency", 1);
+    ctrl_pl.set<bool>("save_grid_data", false);
+    
+    // Add required settings for proper file creation
+    params.set<int>("max_snapshots_per_file", 2);
+    params.set<std::string>("floating_point_precision", "single");
     
     REQUIRE_NOTHROW([&]() {
       OutputManager om;
       om.initialize(comm, params, t0, false);
       om.setup(fm, gm->get_grid_names());
       // OutputManager setup should succeed with alias syntax
+      
+      // Verify that setup completed without errors
+      auto dt = 1;
+      om.init_timestep(t0, dt);
+      om.run(t0 + dt);
+      om.finalize();
     }());
   }
   
-  // Test mixed alias and non-alias field specifications
-  SECTION("mixed_aliases") {
+  // Clean up PIO subsystem
+  scorpio::finalize_subsystem();
+}
+
+// Separate test case with PIO initialization to avoid re-initialization issues
+TEST_CASE("output_aliases_file_verification", "[io][alias]") {
```

Comment:
Why not removing the first TEST_CASE and leave just this one then?

Reply:
Well, only if it adds more coverage. It seems that any coverage that the 1st offers, the 2nd offers it too. Why keeping (and maintaining) the 1st?